src/app/embed/embed.component.css

```diff
@@ -1,7 +1,7 @@
 play-player {
   position: absolute;
   width: 100%;
-  height: 185px;
+  height: 100%;
```

Comment:
Any case for leaving this with some a of max-height of 200px, since it doesn't really expand after that?

...

...

...

(I'm leaning towards "no").